### PR TITLE
Updating esb hiera files for mesos platform

### PR DESCRIPTION
**Pull request by Thanu**
This p/r is to update esb hiera files for mesos platform.

### CODE REVIEW

File: hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml [hunk 1]
```diff
@@ -29,19 +23,24 @@ wso2::clustering:
     marathon_endpoint: http://marathon.mesos:8080/v2/
     mesos_applications: wso2esb-manager,wso2esb-worker
```
Comment (isurulucky) [inline hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml]: Do these hostnames resolve via mesos-dns?

Comment (isurulucky) [inline hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml]: Do these hostnames resolve via mesos-dns?

Comment (imesh) [inline hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml]: Yes Isuru, marathon.mesos points to Mesos master. Others are not hostnames.

Comment (isurulucky) [inline hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml]: Thanks Imesh. What I actually wanted to check was, the hostname and mgt_hostname will resolve. The hostname will be used for internal communication. We might need to check and fix this.

**Pull request merged**